packages/matplotlib/src/browser_backend.py

```diff
@@ -3,7 +3,7 @@
 from matplotlib.backend_bases import FigureCanvasBase, NavigationToolbar2, TimerBase
 
 from js import document
-from pyodide import (
+from pyodide.browser import (
```

Comment:
Maybe call this `js_wrappers` ? It's not really browser-specific, and also exists in node for instance.

Reply:
Maybe this should be a submodule of `ffi` or something. `pyodide.ffi.wrappers`?

Reply:
Yes, sounds good.